packages/office-ui-fabric-react/src/components/Panel/Panel.Props.ts

```diff
@@ -33,11 +33,16 @@ export interface IPanelProps extends React.Props<Panel> {
   headerText?: string;
 
   /**
-  * Event handler for when the panel is closed.
+  * A callback function for when the panel is closed.
```

Comment:
Edit comment to clarify "when the panel will be closed, before the animation starts."